packages/tskit/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: tskit
+  version: 0.4.1
+source:
+  url: https://files.pythonhosted.org/packages/fb/0d/00407181bf409da76dd9a63741d54d57cfa6afec42957ca6a6b93dbfb530/tskit-0.4.1.tar.gz
+  sha256: 28b07fb7f7b19dadd133ab49938c2a38e1558b1cb76122e57f4d1d1230c383ca
+build:
+  script: |
+    pip install -t $HOSTSITEPACKAGES numpy
```

Comment:
Actually, this should not be necessary, as discussed in https://github.com/pyodide/pyodide/pull/2548#discussion_r874083146 so we should revert this line.